src/main/java/com/amazon/opendistroforelasticsearch/sql/rewriter/nestedfield/From.java

```diff
@@ -108,6 +108,12 @@ private void addIfNestedField(Scope scope) {
         Identifier table = new Identifier((SQLIdentifierExpr) ((SQLExprTableSource) expr).getExpr());
         if (table.path().equals(scope.getParentAlias())) {
             scope.addAliasFullPath(emptyIfNull(expr.getAlias()), table.name());
+        } else {
+            String fullPath = scope.getFullPath(table.path());
+
+            if (!fullPath.isEmpty()) {
+                scope.addAliasFullPath(emptyIfNull(expr.getAlias()),fullPath + "." + table.name());
```

Comment:
missing space after comma

Reply:
you got me. will fix it. I am thinking why don't we use any auto-formatting tool. Because when I do gradle build,  It gives me so many warnings, and even a report on style checking. Not only does it contain classes I changed, but also many other classes. That's where I found we are not using any formatting tool.

Reply:
Is it possible to simply this If-else logic with something below
```
            String fullPath = scope.getFullPath(table.path());
            String prefix = fullPath.isEmpty() ? "" : fullPath+".";
            scope.addAliasFullPath(emptyIfNull(expr.getAlias()), prefix+table.name());
```

Reply:
Will simplify if-else logic, thx!